torch/_inductor/graph.py

```diff
@@ -1197,7 +1197,9 @@ def debug(msg):
                 # TODO: this is sus, it probably should be handled in the
                 # lowerings themselves similarly to sym_size/sym-stride
                 debug("is_magic_method")
-                if isinstance(n.meta["val"], torch.SymInt):
+                if isinstance(
+                    n.meta["val"], (torch.SymInt, torch.SymFloat, torch.SymBool)
+                ):
```

Comment:
This one is odd, I got a completely incorrect lowering with operator.truediv was sent to lowerings. This is clearly correct, but it's probably also worth figuring out why we have incorrect lowerings in the lowering dict.

Reply:
tracking issue

Reply:
https://github.com/pytorch/pytorch/issues/127789